pages/serverless-containers/reference-content/containers-sandbox.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: Containers sandbox
+  description: Learn more about the sandboxing environments of Scaleway Serverless Containers.
+content:
+  h1: Containers sandbox
+  paragraph: Learn more about the sandboxing environments of Scaleway Serverless Containers.
+tags: containers sandbox sandboxing gvisor isolation mechanism serverless v1 v2 clock drift skew
+dates:
+  validation: 2025-05-07
+  posted: 2025-05-07
+categories:
+  - serverless
+  - containers
+---
+
+In the context of Scaleway Serverless Containers, a [Sandbox environment](/serverless-containers/concepts/#sandbox) is a critical security mechanism that isolates each container from others, ensuring that they operate in a secure and controlled space. Containers run in dedicated execution environments that prevent unauthorized access and potential interference between different resources.
+
+Scaleway Serverless Containers offers two sandbox environment options:
+
+- Sandbox v1 (legacy)
+- Sandbox v2 (recommended)
+
+## Sandbox v1
+
+Sandbox v1 is Serverless Containers' legacy sandboxing environment with slower [cold starts](/serverless-containers/concepts/#cold-start), but fully supports Linux system call interface. This option is recommended when processing large amounts of *syscalls*.
```

Comment:
```suggestion
Sandbox v1 is Serverless Containers' legacy sandboxing environment with slower [cold starts](/serverless-containers/concepts/#cold-start), but fully supports Linux system call interface.
```

Because when deploying something it's nearly impossible to determine which and how many syscalls you do